src/modules/userType.ts

```diff
@@ -1,5 +1,5 @@
 export default interface UserType {
-  id: number,
+  _id: string,
```

Comment:
reason?

Reply:
Mongo DB ObjectId type is a 24 character string

Reply:
gotcha, thanks!